source/compatibility.tex

```diff
@@ -151,11 +151,11 @@
 \change A name of file scope that is explicitly declared \tcode{const}, and not explicitly
 declared \tcode{extern}, has internal linkage, while in C it would have external linkage
 \rationale
-Because \tcode{const} objects can be used as compile-time values in
+Because \tcode{const} objects may be used as values during translation in
 \Cpp, this feature urges programmers to provide explicit initializer
-values for each \tcode{const}.
-This feature allows the user to put \tcode{const}objects in header files that are included
-in many compilation units.
+for each \tcode{const} object.
+This feature allows the user to put \tcode{const} objects in headers that are included
```

Comment:
Ah, my fault. I should not have missed "source file". I think it is not so much surprisingly (literally) because a "source file" is not a "header" (which is not guaranteed as a "file") or "object/binary/... file" (which is not file of "source" code). It is a common convention that some source files may be provided as the implementation of API exposed by some header files, but actually the latter are also in source code form (and probably in the same source tree), so they are also source files. Perhaps we'd better avoid such daily confusion :) I have already made the rules explicit about _non-header (source) files_ in projects I work on.